src/main/java/teammates/ui/template/InstructorFeedbackSessionActions.java

```diff
@@ -22,10 +22,10 @@
     private String remindParticularStudentsPageLink;
     private String editCopyLink;
 
-    private boolean allowedToEdit;
-    private boolean allowedToDelete;
-    private boolean allowedToSubmit;
-    private boolean allowedToRemind;
+    private boolean isAllowedToEdit;
+    private boolean isAllowedToDelete;
```

Comment:
We need to consider the getter and setter method here.
Currently, `isAllowedToDelelet` corresponds to `public boolean isAllowedToEdit()`. To make it consistent, we shall not change the expression.